spyderlib/config.py

```diff
@@ -724,6 +724,9 @@ def is_ubuntu():
 CONF = UserConfig('spyder', defaults=DEFAULTS, load=True, version=CONF_VERSION,
                   subfolder=SUBFOLDER, backup=True, raw_mode=True)
 
+# Ensures that the config is present on spyder first run
+CONF.set('main', 'interface_language', load_lang_conf())

```

Comment:
@ccordoba12 based on your previous comment I need to add this line here to make sure that on first run of spyder the `interface_language` entry is created. On subsequent starts of spyder that line is harmless so this way is not part of the CONF but ensures the correct behavior. 

You think is ok like this?